### PR TITLE
Added more troubleshooting stuff again

### PR DESCRIPTION
**Pull request by hopolapopola**
I've added some fixes for a couple recent Sileo errors people may encounter to the troubleshooting.md page. I also created a link from using-sileo.md that points to this new section. Let me know if the writing style is okay, and if you'd like some more troubleshooting things e.g. how to figure out which tweak is causing a crash, how to figure out which tweak is causing battery drain etc

### CODE REVIEW

**Review by dabezt31 — CHANGES_REQUESTED** [review]
Good addition honestly, but add a discord.gg/Sileo invite in case people still have issued

**Comment by hopolapopola** [comment]
that's a good idea, I've added a link at the start of the Sileo specific troubleshooting section

File: _pages/en_US/troubleshooting.md [hunk 1]
```diff
@@ -80,3 +80,76 @@ If this doesn't fix it, or there is no update downloaded, you may need to downlo
...
 If this doesn't work, ask on the r/Jailbreak [Discord Server](https://discord.gg/jb) for support.
 {: .notice--info}
+
+## <a name="sileo_errors" /> Fixing package installation errors on Sileo
+
+For the purposes of this troubleshooting section, we are assuming you are running Sileo on Taurine/Odyssey/Odysseyra1n.
+
+In order to determine which error you are encountering, you will have to tap `Show Details` in Sileo when you encounter an error with installing or removing a package.
+
+If the fixes below don't help, feel free to ask on the [Sileo Discord](https://discord.gg/Sileo).
```
Comment (TheMasterOfMike) [inline _pages/en_US/troubleshooting.md:90]: Just saying, for the sake of formatting and consistency, wouldn't it be better to word this as:
> If the fixes below don't help, feel free to ask on **Sileo [Discord Server](https://discord.gg/Sileo) for support**

Comment (hopolapopola) [reply _pages/en_US/troubleshooting.md]: I've changed that sentence so it fits the format better 👍

**Pull request merged**